public/assets/js/scripts.js

```diff
@@ -304,31 +306,45 @@ $(document).ready(function () {
     let spreadFederal = initial >= (avgPropertyCost * 0.01) && (avgPropertyCost <= 15000000) && balance / periodic <= 24 / frequency;
     let rentToOwn = initial >= (avgPropertyCost * 0.05) && balance / periodic <= 120 / frequency;
     let spreadPrivate = initial >= (avgPropertyCost * 0.025) && (avgPropertyCost <= 45000000) && balance / periodic <= 24 / frequency;
+    let hybrid = initial >= (avgPropertyCost * 0.01) && balance / periodic <= 24 / frequency;
     
 
-    if (outrightPersonal) { // 50%
-      paymentType = 'Personal Outright Payment';
-    } else if (immediatePrivate) { // 25% < 45m
-      paymentType = 'Private Immediate Equity';
-    } else if (outrightMortgage) { // 25%
-      paymentType = 'Mortgage Outright Payment';
-    } else if (directSpread) { // 20%
-      paymentType = 'Direct Spread Payment';
-    } else if (immediateFederal) { // 10% < 15m
-      paymentType = 'Federal Immediate Equity';
-    } else if (spreadFederal) { // 1% < 15m
-      paymentType = 'Federal Spread Equity';
-    } else if (spreadPrivate) { // 2.5% < 45m
-      paymentType = 'Private Spread Equity';
-    } else if (rentToOwn) { // 5%
-      paymentType = 'Rent to own package';
-    } else { // 1%
-      paymentType = 'Hybrid Outright Payment';
+    if (outrightPersonal) {
+      output.push('Personal Outright Payment');
```

Comment:
Can you implement the new tabbed design (the updated design we discussed this morning)? Let's not waste our time and energy fixing an outdated design. https://www.figma.com/file/ByKv6q09THdBsd2VV1rRfb/Search-result-page_Dev-team?node-id=260%3A153

![Frame 2](https://user-images.githubusercontent.com/26963369/85312991-7bf2af00-b4af-11ea-8131-4324c2523842.png)